src/main/java/com/sprints/controllers/SprintsController.java

```diff
@@ -1,5 +1,33 @@
 package com.sprints.controllers;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.sprints.exception.EntityNotFoundException;
+import com.sprints.model.Sprint;
+import com.sprints.services.SprintsService;
+
+@RestController
+@RequestMapping(value = "/sprints")
 public class SprintsController {
 
+	@Autowired
+	private SprintsService sprintsService;
+	
+	@DeleteMapping(value = "/{id}")
+	public ResponseEntity<Sprint> deleteSprintById(@PathVariable String id) {
```

Comment:
In the Delete operation you don't need to return a Sprint body.
Use @ResponseStatus and make the method void.

Reply:
Done